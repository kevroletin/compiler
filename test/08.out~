1:1 IDENTIFIER absolute
2:1 OPERATION and
3:1 RESERVED_WORD array
4:1 IDENTIFIER asm
5:1 RESERVED_WORD begin
6:1 RESERVED_WORD case
7:1 RESERVED_WORD const
8:1 IDENTIFIER constructor
9:1 IDENTIFIER destructor
10:1 OPERATION div
11:1 RESERVED_WORD do
12:1 RESERVED_WORD downto
13:1 RESERVED_WORD else
14:1 RESERVED_WORD end
15:1 RESERVED_WORD file
16:1 RESERVED_WORD for
17:1 RESERVED_WORD function
18:1 RESERVED_WORD goto
19:1 RESERVED_WORD if
20:1 IDENTIFIER implementation
21:1 RESERVED_WORD in
22:1 IDENTIFIER ingerited
23:1 IDENTIFIER inline
24:1 IDENTIFIER ifterface
25:1 RESERVED_WORD label
26:1 OPERATION mod
27:1 RESERVED_WORD nil
28:1 RESERVED_WORD not
29:1 IDENTIFIER object
30:1 RESERVED_WORD of
31:1 IDENTIFIER on
32:1 IDENTIFIER operator
33:1 OPERATION or
34:1 RESERVED_WORD packed
35:1 RESERVED_WORD procedure
36:1 RESERVED_WORD program
37:1 RESERVED_WORD record
38:1 IDENTIFIER reintroduce
39:1 RESERVED_WORD repeat
40:1 IDENTIFIER self
41:1 RESERVED_WORD set
42:1 IDENTIFIER shl
43:1 IDENTIFIER shr
44:1 IDENTIFIER string
45:1 RESERVED_WORD then
46:1 RESERVED_WORD to
47:1 RESERVED_WORD type
48:1 IDENTIFIER unit
49:1 IDENTIFIER intil
50:1 IDENTIFIER uses
51:1 RESERVED_WORD var
52:1 RESERVED_WORD while
53:1 RESERVED_WORD with
54:1 OPERATION xor
54:4 END_OF_FILE 