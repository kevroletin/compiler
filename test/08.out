1:1 RESERVED_WORD absolute
2:1 OPERATION and
3:1 RESERVED_WORD array
4:1 RESERVED_WORD asm
5:1 RESERVED_WORD begin
6:1 RESERVED_WORD case
7:1 RESERVED_WORD const
8:1 RESERVED_WORD constructor
9:1 RESERVED_WORD destructor
10:1 OPERATION div
11:1 RESERVED_WORD do
12:1 RESERVED_WORD downto
13:1 RESERVED_WORD else
14:1 RESERVED_WORD end
15:1 RESERVED_WORD file
16:1 RESERVED_WORD for
17:1 RESERVED_WORD function
18:1 RESERVED_WORD goto
19:1 RESERVED_WORD if
20:1 RESERVED_WORD implementation
21:1 RESERVED_WORD in
22:1 RESERVED_WORD inherited
23:1 RESERVED_WORD inline
24:1 RESERVED_WORD interface
25:1 RESERVED_WORD label
26:1 OPERATION mod
27:1 RESERVED_WORD nil
28:1 OPERATION not
29:1 OPERATION object
30:1 RESERVED_WORD of
31:1 RESERVED_WORD on
32:1 RESERVED_WORD operator
33:1 OPERATION or
34:1 RESERVED_WORD packed
35:1 RESERVED_WORD procedure
36:1 RESERVED_WORD program
37:1 RESERVED_WORD record
38:1 RESERVED_WORD reintroduce
39:1 RESERVED_WORD repeat
40:1 RESERVED_WORD self
41:1 RESERVED_WORD set
42:1 OPERATION shl
43:1 OPERATION shr
44:1 RESERVED_WORD string
45:1 RESERVED_WORD then
46:1 RESERVED_WORD to
47:1 RESERVED_WORD type
48:1 RESERVED_WORD unit
49:1 RESERVED_WORD until
50:1 RESERVED_WORD uses
51:1 RESERVED_WORD var
52:1 RESERVED_WORD while
53:1 RESERVED_WORD with
54:1 OPERATION xor
54:4 END_OF_FILE